profile/CSIP.xml

```diff
@@ -193,8 +193,8 @@
             </requirement>
             <requirement ID="CSIP5" REQLEVEL="MAY" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
-                    <head>Other specific content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">When the @csip:CONTENTINFORMATIONTYPE uses the value "OTHER" the @csip:OTHERCONTENTINFORMATIONTYPE must describe the content.</p>
+                    <head>Other detailed specific content type</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">When the @csip:CONTENTINFORMATIONTYPE uses the value "OTHER" the @csip:OTHERCONTENTINFORMATIONTYPE must describe which other specification that is used for describing the content.</p>
```

Comment:
When the @csip:CONTENTINFORMATIONTYPE attribute has the value "OTHER" the @csip:OTHERCONTENTINFORMATIONTYPE attribute must describe the other specification used to describe the content.